OptimizelySDKEventDispatcher/OptimizelySDKEventDispatcher/OPTLYEventDispatcher.m

```diff
@@ -136,7 +136,7 @@ - (void)setupNetworkTimer:(void(^)(void))completion
                                                          repeats:YES];
             
             NSString *logMessage =  [NSString stringWithFormat: OPTLYLoggerMessagesEventDispatcherNetworkTimerEnabled, self.eventDispatcherDispatchInterval];
-            [_logger logMessage:logMessage withLevel:OptimizelyLogLevelDebug];
+            [self->_logger logMessage:logMessage withLevel:OptimizelyLogLevelDebug];
```

Comment:
shouldn't these be weakSelf or at least reference we self and then assign to strong self in block?